site/src/main/tut/Guide.md

```diff
@@ -80,6 +80,26 @@ val renderer = Renderer(
 )
 ```
 
+**For SVG rendering (JVM, Scala.js):**
+
+It is possible to specify for each `RefTree.Ref`
+a hyperlink URL and a tooltip text
+by providing associating to the `RefTree.Ref.id` a URL string
+or a tooltip string respectively.
+
+```scala
+import reftree.render._
+import reftree.diagram._
+
+val renderer = Renderer(
+  renderingOptions = RenderingOptions(
+    density = 75,
+    nodeURLs = Map("1" -> "http://www.example.org", "2" -> "http://www.foobar.org"),
```

Comment:
Another thing, I am not sure I like the idea of attaching this to internal ids, because those ids are not guessable or even deterministic. Is it important to you to specify the tip explicitly? What if reftree generated tips automatically using `toString` or something like that? Or maybe it could just set all tooltips to an empty string.